vimdoc/module.py

```diff
@@ -60,11 +60,17 @@ def Merge(self, block, namespace=None):
       # Overwrite existing section if it's a default.
       if block_id not in self.sections or self.sections[block_id].IsDefault():
```

Comment:
This might be simpler if we just invert that check, unless there's some reason we need to be able to support overwriting a default section with another default section?

```python
# Allow overwriting an existing section if it's a default.
if block_id in self.sections and not self.sections[block_id].IsDefault():
  raise error.DuplicateSection(block_id)
self.sections[block_id] = block
```